gyb_syntax_support/ExprNodes.py

```diff
@@ -374,13 +374,19 @@
              Child('Type', kind='Type'),
          ]),
 
+    Node('ClosureCaptureItemSpecifier', name_for_diagnostics='closure capture specifier', kind='Syntax',
```

Comment:
Are there existing tests for this? If not it would be good to add a few.

Reply:
We’ve got e.g. 
https://github.com/apple/swift-syntax/blob/5545afb01ce3d8e32b1e7b1d4390001625f08d42/Tests/SwiftParserTest/ExpressionTests.swift#L48-L54